app/models/user.rb

```diff
@@ -3,5 +3,5 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-  validates :firstname, :lastname, :email, :password, presence: true
```

Comment:
why remove it?

Reply:
Because this check already have in schema.rb, ` t.string "email", null: false
    t.string "encrypted_password", null: false` 
    And when this validaton was here on spec of valid email and password error.full_messages expected `["Email can't be blank" , "Email can't be blank" ] `